packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -530,8 +531,12 @@ function hasAnyConfiguration(provider: ProviderInfo) {
                 connectionStatus={containerConnectionStatus.get(getProviderConnectionName(provider, container))}
                 updateConnectionStatus={updateContainerStatus}
                 addConnectionToRestartingQueue={addConnectionToRestartingQueue} />
-              <div class="mt-1.5 text-gray-900 text-[9px]" aria-label="Connection Version">
-                <div>{provider.name} {provider.version ? `v${provider.version}` : ''}</div>
+              <div class="mt-1.5 text-gray-900 text-[9px] flex justify-between">
+                <div aria-label="Connection Version">
+                  {provider.name}
+                  {provider.version ? `v${provider.version}` : ''}
+                </div>
+                <div aria-label="Connection Type">{container.vmType ? capitalize(container.vmType) : ''}</div>
```

Comment:
Only capitalize the first letter that looks strange to me ?

Reply:
Would you prefer to have the name lowercase?